recipes/verstr/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "verstr" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/verstr-{{ version }}.tar.gz
+  sha256: 64b47fe4e01778d073796ca76cf8d996a841f15d7c444369e5524f836a5056f7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

Comment:
These should probably also be included: https://github.com/BRGM/verstr/blob/2a0fdd2bc359db94d0730824a07463bf23535e18/pyproject.toml#L2-L6

Reply:
I'm new with conda recipes but I'm wondering if that addition is needed.

What is specified in pyproject.toml [build-system/requires] is only needed by "pip install".
As the build script is basically ["pip install"](https://github.com/fsmaibrgm/verstr-recipe/blob/verstr-recipe/recipes/verstr/meta.yaml#L14), it will create a temporary and isolated environment where the requirements of the pyproject.toml will be fulfilled just for building the wheel.

I have looked at the [pipenv](https://github.com/pypa/pipenv) as an example:
- the recipe : https://github.com/conda-forge/pipenv-feedstock/blob/main/recipe/meta.yaml#L20-30
- the pyproject.toml : https://github.com/pypa/pipenv/blob/main/pyproject.toml#L2
- the setup.py : https://github.com/pypa/pipenv/blob/main/setup.py#L23-28

What I see is that the recipe requirements section only repeats what is needed to run the code, not what pyproject.toml indicated to build the wheel.
I guess the "requirements: host: pip" entry in the recipe is enough to work with the pyproject.toml file.

Reply:
ok, discussing on https://gitter.im/conda-forge/conda-forge.github.io has been enlightening.
I'm going to put those dependencies in the host section.

Reply:
done in 767732eb464ebcb8119228e2591e8f60f96c51d7